src/test/java/webdrivertasks/waitings/WebElementWaiting.java

```diff
@@ -0,0 +1,35 @@
+package webdrivertasks.waitings;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WebElementWaiting {
```

Comment:
`public class WebElementWaitingManager`

from my perspective nouns are better for the class names